eng/common/scripts/git-branch-push.ps1

```diff
@@ -123,9 +121,8 @@ do
                 Write-Error "Unable to create diff file LASTEXITCODE=$($LASTEXITCODE), see command output above."
                 continue
             }
-
-            Write-Host "git reset --hard $RemoteName/${PRBranchName}"
-            git reset --hard $RemoteName/${PRBranchName}
+            Write-Host "git reset --hard origin/${PRBranchName}"
```

Comment:
Why did you change this to origin? It should still be the RemoteName as it was before.